src/modules/45-dashboards/pages/folders/FoldersPage.tsx

```diff
@@ -98,67 +95,57 @@ const NewFolderForm = (props: any): JSX.Element => {
   }
 
   return (
-    <Layout.Horizontal style={{ height: '100%', justifyContent: 'space-between' }}>
-      <Layout.Horizontal padding="xxlarge" style={{ flexDirection: 'column', alignItems: 'baseline' }}>
-        <Heading level={3} font={{ variation: FontVariation.H3 }} padding={{ bottom: 'large' }}>
-          {getString('dashboards.createFolder.stepOne')}
-        </Heading>
-        <Formik
-          initialValues={{ name: '' }}
-          formName={'createFolderForm'}
-          validationSchema={Yup.object().shape({
-            name: Yup.string().trim().required(getString('dashboards.createFolder.folderNameValidation'))
-          })}
-          onSubmit={(formData: { name: string }) => {
-            setErrorMessage('')
-            const response = submitForm(formData)
-            response
-              .then(data => {
-                if (data?.resource) {
-                  history.push({
-                    pathname: routes.toCustomDashboardHome({
-                      folderId: data?.resource,
-                      accountId: accountId
-                    })
+    <Layout.Vertical padding="xxlarge">
+      <Heading level={3} font={{ variation: FontVariation.H3 }} padding={{ bottom: 'large' }}>
+        {getString('dashboards.createFolder.stepOne')}
+      </Heading>
+      <Formik
+        initialValues={{ name: '' }}
+        formName="createFolderForm"
+        validationSchema={Yup.object().shape({
+          name: Yup.string().trim().required(getString('dashboards.createFolder.folderNameValidation'))
+        })}
+        onSubmit={(formData: { name: string }) => {
+          setErrorMessage('')
+          const response = submitForm(formData)
+          response
+            .then(data => {
+              if (data?.resource) {
+                history.push({
+                  pathname: routes.toCustomDashboardHome({
+                    folderId: data?.resource,
+                    accountId: accountId
```

Comment:
Don't change it, but just just a wee FYI, with ES2017+ you simplify things like this
```suggestion
                    accountId
```